app/models/investigation.rb

```diff
@@ -46,7 +46,7 @@ class Investigation < ActiveRecord::Base
     string  :cohorts, :multiple => true do
       cohort_list
     end
-    string  :grade_levels, :multiple => true do
+    string  :grade_level_groups, :multiple => true do
       grade_level_list
```

Comment:
If I understand this right, then the name of this field should still be `grade_levels` because it will the list of grade_levels not the list of grade_level_groups.